### DISCUSSION TITLE
Why does `js` not find the div with the given id?

### DISCUSSION THREAD

**Discussion by axiomtutor** [pyscript/pyscript#1646] [Q&A]
I don't know if this is an issue or merely a question to ask (I guess it depends on whether I'm making a mistake or some piece of software is!).  

But the long-story-short is that I made a div element with an id, then tried to access it through js inside of a PyScript script, and js doesn't find the element (and appears to add a `#` to the id).  

The full description of the code that I wrote and the error that I get is detailed here:

https://stackoverflow.com/questions/76940340/pyscript-cant-find-the-element-id

I haven't had an answer after about a day -- is that some sign that this is an issue with the software?  Should I raise this as an issue here?

**Comment by axiomtutor** [comment]
Never mind, I just realized I didn't know JS well enough.  My problem was that I can't start an id with a number.

**Reply by WebReflection** [reply]
don't be harsh on you ... many don't know and that's not even a JS thing, it's a W3C / WHATWG specification about DOM ids.
